ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -27,6 +27,8 @@ let net_cmd = !Xapi_globs.net_cmd
 
 let tdb_tool = !Xapi_globs.tdb_tool
 
+let debug_level = "10"
```

Comment:
If we make this a binding, it should have type `int`.

Reply:
Constants like this should go into xapi_globs.ml, and made configurable from xapi.conf by adding them to `xapi_globs_spec` near the bottom of that file.